src/Util/Test.php

```diff
@@ -517,7 +518,7 @@ private static function getDataFromDataProviderAnnotation($docComment, $classNam
                     $data = $dataProviderMethod->invoke($object, $methodName);
                 }
 
-                if ($data instanceof Iterator) {
+                if ($data instanceof Traversable) {
                     $data = \iterator_to_array($data);
```

Comment:
that's why I lowercased it, didn't I?
is_iterable is polyfilled even in 5.x

https://github.com/symfony/polyfill/tree/master/src/Php71